src/main/java/ClosestSubwayStations.java

```diff
@@ -0,0 +1,32 @@
+import java.util.Arrays;
+import java.util.List;
+
+public class ClosestSubwayStations {
+
+    public double calculateDistanceFormula (List<Double> locationCoords, List<Double> stationCoords) {
+        double calculateLatitude = Math.pow((stationCoords.get(0) - locationCoords.get(0)), 2);
+        double calculateLongitude = Math.pow((stationCoords.get(1) - locationCoords.get(1)), 2);
+        return Math.sqrt(calculateLatitude + calculateLongitude);
+    }
+
+    public List<SubwayStation.Station> calculateClosestStations (List<Double> originCoords, List<Double> destinationCoords, List<SubwayStation.Station> subwayStation) {
```

Comment:
Write a `getClosestStation(List<Double> coords)` method and just call it twice, it's simpler.